test/e2e/catalog_e2e_test.go

```diff
@@ -242,10 +242,10 @@ var _ = Describe("Catalog", func() {
 		}
 
 		// Create the initial catalogsource
-		createInternalCatalogSource(GinkgoT(), c, crc, mainCatalogName, testNamespace, mainManifests, nil, []v1alpha1.ClusterServiceVersion{mainCSV})
+		createInternalCatalogSource(c, crc, mainCatalogName, testNamespace, mainManifests, nil, []v1alpha1.ClusterServiceVersion{mainCSV})
```

Comment:
This method uses deprecated `internal` catalog source type. I was thinking of rewriting this method to use `configMap` catalog source type as a separate PR. Thoughts?